CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 0.12.21
```

Comment:
0.12.22

Also at some point (not necessarily this CL), please add changelog entries for the `retry` option in its various forms.